### PR TITLE
Add MetricsRegisterer to oracle args

### PR DESCRIPTION
**Pull request by george-dorin**
- Bump libocr version
- Add MetricsRegisterer

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,35 +9,41 @@ replace (
...
-	github.com/ethereum/go-ethereum v1.12.0
+	github.com/ethereum/go-ethereum v1.13.8
 	github.com/go-echarts/go-echarts/v2 v2.2.6
 	github.com/google/uuid v1.3.1
 	github.com/jedib0t/go-pretty/v6 v6.4.7
 	github.com/pkg/errors v0.9.1
 	github.com/shopspring/decimal v1.3.1
 	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240118012339-4864e2306bb1
-	github.com/smartcontractkit/libocr v0.0.0-20230925165524-ffa38fe11ef8
+	github.com/smartcontractkit/libocr v0.0.0-20240229181116-bfb2432a7a66
```
Comment (infiloop2) [inline go.mod:19]: there's a newer commit `5a9177221bec36ac3bf76a161c81a3389d5b17b6`  in libocr master. Do we want to target that?

File: pkg/v2/delegate.go [hunk 2]
```diff
@@ -36,6 +37,7 @@ type DelegateConfig struct {
 	OffchainKeyring              types.OffchainKeyring
 	OnchainKeyring               types.OnchainKeyring
 	LocalConfig                  types.LocalConfig
+	MetricsRegisterer            prometheus.Registerer
```
Comment (infiloop2) [inline pkg/v2/delegate.go:40]: Could you create a corresponding PR in core which populate this?

**Review by infiloop2 — CHANGES_REQUESTED** [review]
Requesting changes for couple of questions
- Whether we want to upgrade to latest libocr
- Ask for corresponding PR in code which uses this

File: pkg/v2/delegate.go (hunk 2 above)
Comment (george-dorin) [reply pkg/v2/delegate.go:40]: @infiloop2 Yeah, I'll create the corresponding PR

**Comment by george-dorin** [comment]
Corresponding PR: https://github.com/smartcontractkit/chainlink/pull/12359

**Review by infiloop2 — APPROVED** [review]

**Pull request merged**